source/style-guide/blog-guidelines.txt

```diff
@@ -1,209 +1,78 @@
 .. _blog-guidelines:
 
 ===============
-Blog guidelines
+Blog Guidelines
 ===============
 
-Developers often ask an Technical Writer to edit a blog that they
-have written and placed in the
-https://github.com/rackerlabs/docs-developer-blog repository. Blogs in
-this repository are published to the MongoDB Developer Blog site at
-https://developer.MongoDB.com/blog/.
+When you write a blog, your grammar should be correct but you do not
```

Comment:
This may be old-fashioned, but I think "blog post" or "blog article" would be more appropriate here. To me a "blog" is made up of many "blog posts", so you "write a blog post" but "maintain a blog". Is that hopelessly 1997 of me?

Reply:
Not at all. ;) Updated.